static-pages/speakers.md

```diff
@@ -0,0 +1,107 @@
+---
+title: SyDjango Speaker Resources
+---
+
+# SyDjango Speaker Resources
+
+## Speak at SyDjango
+
+You've got an awesome idea for a talk and want to present it at one of the future SyDjango events?
+That's great! We're excited to have you present at our meetup. On this page we provide you with all
+the information you need to prepare for your talk.
+
+First of all, please submit this form: https://sydjango.org/propose-talk. Once done SyDjango org
+team will be notified and one of us will be in touch to confirm the date and the rest of details.
+Thank you for volunteering!
+
+Next, start preparing for your talk to be ready on the scheduled date. Please, keep reading further
+for some advices.
+
+Finally, sometimes things don't go as planned. We understand this. If situation changed on your side
+and you can't anymore to present on the scheduled date, please try to let us know as soon as you can
+and we will be happy to reschedule your talk.
+
+Don't hesitate to ask questions!
+
+## Talks format
+
+We are broadly trying to copy the same talks format as PyCon Australia. The general format is as
+follows:
+
+- Regular talks: 30 minutes. We recommend to aim for 20 min talk + 10 min Q&A.
+- Lightning talks: 5 minutes. Slides are encouraged but not required.
+
+## Topics
+
+The incomplete list of high-level topics the group members are interested is below:
+
+– OSS projects related to Django – APIs – Backend Software Engineering – Frontend Software
+Engineering – Infrastructure – DevOps – Site Reliability Engineering – Scalability – Security – QA –
+Data – ML/AI – Soft Skills – Career in Tech – Startups
+
+We are happy to accept talks that are at least loosely connected to Django. Of course, the stronger
+the connection is the better.
+
+## Can you help me to prepare for my talk?
+
+Sure, please find below some common sense advice for preparing your talk.
+
+### Mind your target audience
+
+We have engineers of all level attending our events. It's usually a good ideal to assume the minimal
+knowledge on the topic from the audience. It doesn't matter how advanced your topic is please try to
+make it accessible to everyone.
+
+### Keep it simple
+
+Please skip unnecessary details. The simpler your material is the easier it is to follow and
+comprehend.
+
+### Be concise
+
+Use simple language. Avoid jargon unless it's not an essential part of your talk.
+
+### Work on connecting parts of your talks
+
+It's always easier to follow the material when its logical parts are well connected. Ideally, try to
+make your presentation to sound like a story.
+
+### Use visuals
+
+Visuals can supplement your presentation very well. It's worth nothing to say it helps to make your
+points more easily understood.
```

Comment:
"It's worth nothing to say it helps to make your points more easily understood."
ParseError :)
suggest just: it helps to make your
points more easily understood.